### PR TITLE
docs: more whitespace, wording, and responsiveness changes

### PR DESCRIPTION
**Pull request by tgerla**
- tweak whitespace between sections
- fix the top menu for small screens
- fix the terminal overlapping on small screens
- tweak wording on a few of the bullet points
- clean up the display of the "certified" logo on small screens
- clean up the "features" grid on medium/large screens

Signed-off-by: Tim Gerla <tim@gerla.net>

### CODE REVIEW

File: docs/website/pages/index.vue [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
       <div class="flex flex-wrap justify-center items-center">
-        <div class="px-4 md:px-2 md:w-3/4">
+        <div class="px-4 md:px-2 md:w-3/4 text-xl text-gray-700">
           <p>
             Talos is a modern OS designed to be secure, immutable, and minimal.
             Its purpose is to host Kubernetes clusters, so it is tightly
             integrated with Kubernetes.
           </p>
-          <p>
+          <p class="md:text-base">
```
Comment (andrewrynhard) [inline docs/website/pages/index.vue]: The text above this looks bigger. Is that intentional. Looks a little off to me.

File: docs/website/pages/index.vue [hunk 2]
```diff
@@ -64,17 +63,17 @@
       </div>
     </div>
 
-    <div class="w-3/4 mx-auto mb-20">
+    <div class="w-3/4 mx-auto mb-20 pt-4">
       <div class="flex flex-col justify-center">
         <div class="text-center">
-          <h2>API Driven</h2>
+          <h2>API Control</h2>
```
Comment (andrewrynhard) [inline docs/website/pages/index.vue]: I don't know if "API Control" sounds good. I don't like "API Driven" neither though.

File: docs/website/pages/index.vue (hunk 1 above)
Comment (tgerla) [reply docs/website/pages/index.vue]: Yeah, it was intentional. Does this look too big all around? I thought the regular font was a bit small...

Screen Shot 2019-10-28 at 11 42 50 AM

File: docs/website/pages/index.vue (hunk 2 above)
Comment (tgerla) [reply docs/website/pages/index.vue]: Hmm...what about one of these:

"Modern CLI and API"
"API and CLI Driven"
"Automation-Ready CLI and API"

Something like that...it occurs to me that we are talking about an API and then showing a CLI, and maybe we should be explicit about it? Not sure!

File: docs/website/pages/index.vue (hunk 1 above)
Comment (andrewrynhard) [reply docs/website/pages/index.vue]: On my screen, the second paragraph looks smaller.

Screen Shot 2019-10-28 at 8 46 56 AM

File: docs/website/pages/index.vue (hunk 2 above)
Comment (andrewrynhard) [reply docs/website/pages/index.vue]: I don't know the best title. We could drop it really and just have the terminal. We could then add a feature bullet point around the API.

Comment (tgerla) [reply docs/website/pages/index.vue]: Yeah, that's a good idea.

File: docs/website/pages/index.vue (hunk 1 above)
Comment (tgerla) [reply docs/website/pages/index.vue]: Sorry, yes, my screenshot was a mockup of making both paragraphs the larger size. The PR has the first paragraph bigger. I agree that it looks a bit off. Do you like the bigger font for both paragraphs?

Comment (andrewrynhard) [reply docs/website/pages/index.vue]: I do.

**Pull request merged**